site/en/blog/storage-buckets/index.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: 'layouts/blog-post.njk'
+title: 'Not all storage is created equal: introducing Storage Buckets'
+subtitle: >
+  When the browser is under heavy storage pressure, it can evict your least important storage
```

Comment:
this subtitle description is a bit odd, it's a problem statement. Maybe, "We're experimenting with an API for persistent storage, and we'd love your feedback."

Reply:
What do you think of this suggestion? https://github.com/GoogleChrome/developer.chrome.com/pull/4139/files#r1013974736